calicoctl/commands/node/run.go

```diff
@@ -259,16 +258,6 @@ Description:
 		return fmt.Errorf("Error executing command: invalid to disable Docker Networking and enable Container labels")
 	}
 
-	// Set CALICO_LIBNETWORK_IFPREFIX env variable if Docker network is enabled and set to non-default value.
-	if !disableDockerNw && ifprefix != DEFAULT_DOCKER_IFPREFIX {
```

Comment:
Can we remove the `disableDockerNw` var too?

Reply:
Thank you for your suggestion. I double-checked it with Casey and it can indeed be removed. I did this in a new commit.